modin/config/envvars.py

```diff
@@ -99,16 +99,29 @@ def _get_default(cls):
             import distributed
 
         except ImportError:
-            raise ImportError(
-                "Please `pip install modin[ray]` or `modin[dask]` to install an engine"
-            )
-        if version.parse(dask.__version__) < version.parse("2.22.0") or version.parse(
-            distributed.__version__
-        ) < version.parse("2.22.0"):
-            raise ImportError(
-                "Please `pip install modin[dask]` to install compatible Dask version."
-            )
-        return "Dask"
+            pass
+        else:
+            if version.parse(dask.__version__) < version.parse(
+                "2.22.0"
+            ) or version.parse(distributed.__version__) < version.parse("2.22.0"):
+                raise ImportError(
+                    "Please `pip install modin[dask]` to install compatible Dask version."
+                )
+            return "Dask"
+        try:
+            import omniscidbe  # noqa
+        except ImportError:
+            try:
+                import dbe  # noqa
+            except ValueError:
+                pass
```

Comment:
why it's catching `ValueError` here, shouldn't it be an `ImportError`?

Reply:
Changed.